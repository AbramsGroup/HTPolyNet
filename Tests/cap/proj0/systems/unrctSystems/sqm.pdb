ATOM      1  O   FDE     1       0.937   2.330   0.513  1.00  0.00           O
ATOM      2  C   FDE     1       0.247   2.636   1.685  1.00  0.00           C
ATOM      3  C3  FDE     1      -0.014   1.485   2.401  1.00  0.00           C
ATOM      4  C4  FDE     1       1.104   0.937   0.502  1.00  0.00           C
ATOM      5  C5  FDE     1       0.540   0.388   1.637  1.00  0.00           C
ATOM      6  C6  FDE     1       1.778   0.366  -0.690  1.00  0.00           C
ATOM      7  N   FDE     1       3.218   0.201  -0.521  1.00  0.00           N
ATOM      8  H   FDE     1       1.536   1.021  -1.581  1.00  0.00           H
ATOM      9  H1  FDE     1       1.331  -0.656  -0.875  1.00  0.00           H
ATOM     10  C7  FDE     1       3.933   1.467  -0.464  1.00  0.00           C
ATOM     11  C8  FDE     1       3.785  -0.717  -1.501  1.00  0.00           C
ATOM     12  C9  FDE     1       3.486  -2.199  -1.196  1.00  0.00           C
ATOM     13  H2  FDE     1       4.904  -0.591  -1.495  1.00  0.00           H
ATOM     14  H3  FDE     1       3.416  -0.498  -2.545  1.00  0.00           H
ATOM     15  C1  FDE     1       4.556  -3.084  -1.811  1.00  0.00           C
ATOM     16  H4  FDE     1       4.634  -2.895  -2.909  1.00  0.00           H
ATOM     17  H5  FDE     1       4.302  -4.160  -1.655  1.00  0.00           H
ATOM     18  H6  FDE     1       5.545  -2.878  -1.337  1.00  0.00           H
ATOM     19  O1  FDE     1       2.213  -2.477  -1.772  1.00  0.00           O
ATOM     20  H7  FDE     1       3.435  -2.381  -0.086  1.00  0.00           H
ATOM     21  C10 FDE     1       5.236   1.397   0.360  1.00  0.00           C
ATOM     22  H8  FDE     1       3.264   2.239   0.013  1.00  0.00           H
ATOM     23  H9  FDE     1       4.202   1.845  -1.494  1.00  0.00           H
ATOM     24  O2  FDE     1       6.024   2.540   0.061  1.00  0.00           O
ATOM     25  C2  FDE     1       4.956   1.294   1.845  1.00  0.00           C
ATOM     26  H10 FDE     1       5.882   0.536   0.025  1.00  0.00           H
ATOM     27  H11 FDE     1       4.342   2.161   2.191  1.00  0.00           H
ATOM     28  H12 FDE     1       5.916   1.291   2.414  1.00  0.00           H
ATOM     29  H13 FDE     1       4.398   0.353   2.064  1.00  0.00           H
ATOM     30  H14 FDE     1       5.549   3.311   0.392  1.00  0.00           H
ATOM     31  H15 FDE     1       2.042  -3.416  -1.637  1.00  0.00           H
ATOM     32  H16 FDE     1       0.041   3.693   1.822  1.00  0.00           H
ATOM     33  H17 FDE     1      -0.533   1.397   3.351  1.00  0.00           H
ATOM     34  H18 FDE     1       0.511  -0.660   1.918  1.00  0.00           H
