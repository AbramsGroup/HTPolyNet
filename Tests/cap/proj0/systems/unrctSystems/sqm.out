            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 43

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       79
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.030775 seconds
| QMMM:                 Dspev diag routine = 0.034853 seconds
| QMMM:                Dspevd diag routine = 0.012102 seconds
| QMMM:                Dspevx diag routine = 0.054024 seconds
| QMMM:                 Dsyev diag routine = 0.043208 seconds
| QMMM:                Dsyevd diag routine = 0.072562 seconds
| QMMM:                Dsyevr diag routine = 0.030759 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.011142 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O       -0.1470    1.3670   -0.2320
  QMMM:     2        2      C       -0.9000    1.0270    0.7890
  QMMM:     3        3      C       -0.1260    0.4150    1.7530
  QMMM:     4        4      C        1.0910    1.0020    0.0150
  QMMM:     5        5      C        1.1580    0.4000    1.2590
  QMMM:     6        6      C        2.2230    1.2420   -0.9450
  QMMM:     7        7      N        3.5020    0.6910   -0.4220
  QMMM:     8        8      H        2.2590    2.3440   -1.0860
  QMMM:     9        9      H        1.9750    0.8210   -1.9440
  QMMM:    10       10      C        4.5900    1.6930   -0.5680
  QMMM:    11       11      C        3.8610   -0.6060   -1.0640
  QMMM:    12       12      C        2.9150   -1.7740   -0.7000
  QMMM:    13       13      H        4.8570   -0.9270   -0.7060
  QMMM:    14       14      H        3.9310   -0.4910   -2.1700
  QMMM:    15       15      C        3.4590   -3.0940   -1.2540
  QMMM:    16       16      H        3.5280   -3.0510   -2.3620
  QMMM:    17       17      H        2.7840   -3.9290   -0.9700
  QMMM:    18       18      H        4.4670   -3.3040   -0.8370
  QMMM:    19       19      O        1.6330   -1.5620   -1.2310
  QMMM:    20       20      H        2.8730   -1.8470    0.4110
  QMMM:    21       21      C        5.8390    1.3870    0.2950
  QMMM:    22       22      H        4.2260    2.6860   -0.2170
  QMMM:    23       23      H        4.8800    1.8000   -1.6380
  QMMM:    24       24      O        6.7970    2.4030    0.1310
  QMMM:    25       25      C        5.4880    1.2210    1.7790
  QMMM:    26       26      H        6.3210    0.4490   -0.0420
  QMMM:    27       27      H        5.0220    2.1470    2.1790
  QMMM:    28       28      H        6.4100    1.0040    2.3610
  QMMM:    29       29      H        4.7820    0.3750    1.9190
  QMMM:    30       30      H        6.4130    3.2470    0.4890
  QMMM:    31       31      H        0.9990   -2.0890   -0.6780
  QMMM:    32       32      H       -1.9650    1.2070    0.8510
  QMMM:    33       33      H       -0.4610    0.0270    2.7050
  QMMM:    34       34      H        2.0320   -0.0020    1.7530

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -101.6117 kcal/mol        1.0359 kcal/(mol*A)
xmin    20     -102.8565 kcal/mol        2.0586 kcal/(mol*A)
xmin    30     -103.5306 kcal/mol        0.4056 kcal/(mol*A)
xmin    40     -103.7319 kcal/mol        0.3665 kcal/(mol*A)
xmin    50     -104.1965 kcal/mol        0.3673 kcal/(mol*A)
xmin    60     -104.5692 kcal/mol        0.1335 kcal/(mol*A)
xmin    70     -104.7005 kcal/mol        0.1237 kcal/(mol*A)
xmin    80     -104.7413 kcal/mol        0.0822 kcal/(mol*A)
xmin    90     -104.8006 kcal/mol        0.0890 kcal/(mol*A)
xmin   100     -104.8295 kcal/mol        0.1818 kcal/(mol*A)
xmin   110     -104.8543 kcal/mol        0.1422 kcal/(mol*A)
xmin   120     -104.9133 kcal/mol        0.3321 kcal/(mol*A)
xmin   130     -105.0390 kcal/mol        0.3453 kcal/(mol*A)
xmin   140     -105.0947 kcal/mol        0.0723 kcal/(mol*A)
xmin   150     -105.1180 kcal/mol        0.0658 kcal/(mol*A)
xmin   160     -105.1217 kcal/mol        0.0206 kcal/(mol*A)
xmin   170     -105.1263 kcal/mol        0.0142 kcal/(mol*A)
xmin   180     -105.1275 kcal/mol        0.0148 kcal/(mol*A)
xmin   190     -105.1277 kcal/mol        0.0141 kcal/(mol*A)
xmin   200     -105.1281 kcal/mol        0.0038 kcal/(mol*A)
xmin   210     -105.1281 kcal/mol        0.0021 kcal/(mol*A)
xmin   220     -105.1281 kcal/mol        0.0006 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.6049  -39.6825  -38.5848  -36.9808  -33.2613  -32.4656  -31.9258  -30.4443
 -27.6688  -26.8946  -24.2839  -22.3799  -22.1143  -21.6370  -20.7272  -18.4405
 -17.7465  -17.5387  -16.9692  -16.6569  -16.2196  -15.7218  -15.4428  -15.1256
 -14.6746  -14.4129  -13.9573  -13.7762  -13.7403  -13.5512  -13.4172  -12.9622
 -12.8499  -12.6431  -12.5887  -12.4777  -11.9454  -11.4939  -11.0994  -11.0395
 -10.5547   -9.3332   -9.0595    0.6651    1.6855    2.4531    2.8085    3.3153
   3.5412    3.6109    3.7107    3.8632    3.8779    3.9371    4.0217    4.0534
   4.0868    4.1357    4.2368    4.3302    4.4287    4.4759    4.5662    4.6179
   4.6604    4.7832    4.8381    4.9016    4.9038    4.9956    5.0482    5.1502
   5.2430    5.7132    5.9798    6.0525    6.1240    6.2039    6.4391

 Heat of formation   =       -105.12813664 kcal/mol  (       -4.55869809 eV)

 Total SCF energy    =     -65443.88152779 kcal/mol  (    -2837.85965603 eV)
 Electronic energy   =    -395444.56326960 kcal/mol  (   -17147.76303151 eV)
 Core-core repulsion =     330000.68174181 kcal/mol  (    14309.90337547 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      O                 -0.115
     2      C                 -0.098
     3      C                 -0.200
     4      C                 -0.041
     5      C                 -0.179
     6      C                 -0.010
     7      N                 -0.260
     8      H                  0.078
     9      H                  0.136
    10      C                 -0.106
    11      C                 -0.062
    12      C                  0.023
    13      H                  0.097
    14      H                  0.078
    15      C                 -0.250
    16      H                  0.088
    17      H                  0.076
    18      H                  0.087
    19      O                 -0.338
    20      H                  0.091
    21      C                  0.028
    22      H                  0.102
    23      H                  0.071
    24      O                 -0.329
    25      C                 -0.240
    26      H                  0.112
    27      H                  0.074
    28      H                  0.081
    29      H                  0.100
    30      H                  0.196
    31      H                  0.200
    32      H                  0.187
    33      H                  0.160
    34      H                  0.162
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -0.587   -1.359    0.649    1.616
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O        0.9367    2.3299    0.5130
  QMMM:     2        2      C        0.2467    2.6360    1.6855
  QMMM:     3        3      C       -0.0142    1.4853    2.4005
  QMMM:     4        4      C        1.1038    0.9371    0.5022
  QMMM:     5        5      C        0.5405    0.3883    1.6375
  QMMM:     6        6      C        1.7778    0.3665   -0.6901
  QMMM:     7        7      N        3.2182    0.2007   -0.5215
  QMMM:     8        8      H        1.5365    1.0208   -1.5811
  QMMM:     9        9      H        1.3314   -0.6558   -0.8753
  QMMM:    10       10      C        3.9326    1.4668   -0.4640
  QMMM:    11       11      C        3.7847   -0.7169   -1.5009
  QMMM:    12       12      C        3.4861   -2.1994   -1.1965
  QMMM:    13       13      H        4.9042   -0.5914   -1.4950
  QMMM:    14       14      H        3.4156   -0.4978   -2.5454
  QMMM:    15       15      C        4.5565   -3.0836   -1.8113
  QMMM:    16       16      H        4.6344   -2.8950   -2.9087
  QMMM:    17       17      H        4.3022   -4.1595   -1.6551
  QMMM:    18       18      H        5.5450   -2.8780   -1.3370
  QMMM:    19       19      O        2.2128   -2.4765   -1.7719
  QMMM:    20       20      H        3.4349   -2.3808   -0.0859
  QMMM:    21       21      C        5.2363    1.3965    0.3604
  QMMM:    22       22      H        3.2635    2.2393    0.0129
  QMMM:    23       23      H        4.2019    1.8452   -1.4940
  QMMM:    24       24      O        6.0240    2.5399    0.0607
  QMMM:    25       25      C        4.9557    1.2943    1.8454
  QMMM:    26       26      H        5.8821    0.5364    0.0247
  QMMM:    27       27      H        4.3423    2.1611    2.1908
  QMMM:    28       28      H        5.9161    1.2912    2.4145
  QMMM:    29       29      H        4.3977    0.3533    2.0637
  QMMM:    30       30      H        5.5494    3.3113    0.3922
  QMMM:    31       31      H        2.0419   -3.4155   -1.6373
  QMMM:    32       32      H        0.0407    3.6929    1.8219
  QMMM:    33       33      H       -0.5326    1.3970    3.3506
  QMMM:    34       34      H        0.5106   -0.6604    1.9176

           --------- Calculation Completed ----------

